### ISSUE TITLE
Select2 choices?

### ISSUE THREAD

**Issue by samsebastian** [kaliber5/ember-bootstrap#42]
In the docs for select box items I can see the following: 
```
{{bs-form-element controlType="select" choices=countries choiceLabelProperty="name" choiceValueProperty="id" label="Country" value=selectedCountry}}
```
The above works fine when using `controlType="select"` but when you change to `controlType="select2"` there are no options to select from?

I noticed select2 is a valid option in:
```
The type of the control widget. Supported types:

'text'
'checkbox'
'select'
'select2' (needs ember-select-2)
'textarea'
any other type will use an input tag with the controlType value as the type attribute (for e.g. HTML5 input types like 'email'), and the same layout as the 'text' type
```

and I have run `ember install ember-select-2` so I'm just wondering if there is a different way to load the choices in if using select2?

**Comment by simonihmig** [comment]
Indeed, support is incomplete. With a horizontal form layout it should work just like a regular select, but the other form layouts are missing. I will however probably drop direct support for select2 and instead provide a way to use arbitrary custom controls.

**Comment by simonihmig** [comment]
@samsebastian See the just released v0.6.1 for ways to use custom controls: http://kaliber5.github.io/ember-bootstrap/api/classes/Components.FormGroup.html
